utils/loading_cache.hh

```diff
@@ -52,6 +52,7 @@ struct simple_entry_size {
 struct do_nothing_loading_cache_stats {
     // Accounts events when entries are evicted from the unprivileged cache section due to size restriction.
     // These events are interesting because they are an indication of a cache pollution event.
+    static void inc_privileged_on_cache_size_eviction() noexcept {};
```

Comment:
The comment above refers and explains the "unprivileged" counter, so should keep the comment above the right counter.
It would be great (but not necessary) if you write an explanation of why the "privileged" counter is also interesting because this comment suggests that only the unprivileged one is interesting. To be honest, I have no idea what the answer to this is... Maybe https://github.com/scylladb/scylladb/issues/10463 has an explanation of why the privileged counter is interesting? Maybe @avelanarius or @vladzcloudius can help explain.

Reply:
I think the main motivation for this new metric is that currently you can't get a full number of cache evictions - only `authorized_prepared_statements_unprivileged_entries_evictions_on_size` and the privileged evictions are not counted.

Alternatively, we could delete that metric and introduce a metric counting a total number of evictions (privileged + unprivileged), but that wouldn't be backwards compatible.

Reply:
Done.